.devcontainer/devcontainer.json

```diff
@@ -1,20 +1,50 @@
 {
-	"name": "Immich devcontainers",
-	"build": {
-		"dockerfile": "Dockerfile",
-		"args": {
-			"BASEIMAGE": "mcr.microsoft.com/devcontainers/typescript-node:22"
-		}
-	},
+	"name": "Immich",
+  "service": "immich-devcontainer",
+  "dockerComposeFile": [
+    "docker-compose.yml",
+    "../docker/docker-compose.dev.yml"
+  ],
 	"customizations": {
 		"vscode": {
+      "settings": {
```

Comment:
I think we already have this in .vscode/settings.json. Does it automatically pick that up already, or do we have to include it here as well? If the latter, is there a way to do it as an import instead?